src/locale.js

```diff
@@ -146,3 +184,85 @@ export default class Locale {
 function getLanguageDirection( languageCode ) {
 	return RTL_LANGUAGE_CODES.includes( languageCode ) ? 'rtl' : 'ltr';
 }
+
+/**
+ * A translated message returned from one of the {@link Locale}'s methods.
+ */
+export class Message {
+	/**
+	 * @hidden
```

Comment:
What's the `hidden` tag? I'm afraid we don't use it anywhere. And I couldn't find it in [https://jsdoc.app/](https://jsdoc.app/)  too.